service-print/src/main/java/org/oskari/print/request/PDPrintStyle.java

```diff
@@ -108,4 +119,19 @@ public boolean hasLineColor () {
     public boolean hasLabels () {
         return labelProperty != null && !labelProperty.isEmpty();
     }
+
+    public static class LabelAlign {
+        private float x;
```

Comment:
These are offsets, right? Should they be called `offsetX` and `offsetY` instead of `x` and `y`?